### PR TITLE
mkcloud: add support for deploying designate

### PR DESCRIPTION
**Pull request by dirkmueller**
Adds a want_designate_proposal option that defaults to false.

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -3131,7 +3142,7 @@ function onadmin_proposal
     safely oncontroller check_crm_failcounts
     # For all remaining proposals, check for HA failures after each deployment
     for proposal in horizon ceilometer heat manila trove \
-        barbican magnum sahara murano aodh tempest; do
+        designate barbican magnum sahara murano aodh tempest; do
```
Comment (toabctl) [inline scripts/qa_crowbarsetup.sh]: Don't we propose to deploy the barclamps from the list from top to bottom? then (assuming https://github.com/crowbar/crowbar-openstack/pull/1707 gets merged), designate would come after cinder

Comment (toabctl) [reply scripts/qa_crowbarsetup.sh]: No. Cinder does **not** depend on Designate. I thought we recommend to deploy the barclamps in the given order (you can skip barlamps, of course). But I can't find that documentation so just ignore my comment.

Comment (nicolasbock) [reply scripts/qa_crowbarsetup.sh]: Ah ok :smile:

Comment (dirkmueller) [reply scripts/qa_crowbarsetup.sh]: @toabctl I think this list is mostly unsorted. I can deploy it earlier if you like

Comment (toabctl) [reply scripts/qa_crowbarsetup.sh]: if it doesn't matter we can leave it as it is.

**Comment by bmwiedemann** [comment]
SOC9 mkcloud failed in designate

**Comment by toabctl** [comment]
> we need to add a line in scripts/lib/mkcloud-common.sh next to `: ${want_magnum_proposal:=0}`

@bmwiedemann done

**Comment by toabctl** [comment]
- @dirkmueller is there a reason why this is still WIP?
- @bmwiedemann @nicolasbock could you have another look please?

**Comment by dirkmueller** [comment]
@toabctl I wanted to test it at least once..

**Pull request merged**